js/templates/modals/listingDetail/listing.html

```diff
@@ -2,11 +2,11 @@
 
 <div class="topControls gutterHSm flex">
   <% if (ob.vendor) { %>
-  <div class="contentBox clrP clrBr clrT">
-    <div class="padSm gutterHSm overflowAuto flexVCent">
+  <div class="contentBox clrP clrSh3 clrBr clrT">
+    <div class="padSm gutterHSm overflowAuto margRSm flexVCent">
       <a class="clrBr2 clrSh1 disc storeOwnerAvatar flexNoShrink js-storeOwnerAvatar" style="<%= ob.getAvatarBgImage(ob.vendor.avatar) %>"></a>
-      <p class="txUnl clamp"><%= ob.vendor.name %></p>
-      <a class="link flexNoShrink js-goToStore"><% print(ob.openedFromStore ? ob.polyT('listingDetail.returnToStore'): ob.polyT('listingDetail.goToStore')) %></a>
+      <p class="txUnl tx3 clamp"><%= ob.vendor.name %></p>
+      <a class="link flexNoShrink tx6 margRSm js-goToStore"><% print(ob.openedFromStore ? ob.polyT('listingDetail.returnToStore'): ob.polyT('listingDetail.goToStore')) %></a>
```

Comment:
It adds a bit of margin to the right of the container
![screenshot 2017-01-31 13 13 07](https://cloud.githubusercontent.com/assets/13066886/22484605/14928c54-e7b7-11e6-8e23-977eb7991ed8.png)

Without it...
![screenshot 2017-01-31 13 13 16](https://cloud.githubusercontent.com/assets/13066886/22484606/14abec3a-e7b7-11e6-9970-4a49871113e8.png)